lib/mayaUsd/render/vp2RenderDelegate/mayaPrimCommon.cpp

```diff
@@ -177,21 +177,20 @@ void MayaUsdRPrim::_UpdateTransform(
     }
 
     // Local-to-world transformation
-    MMatrix& worldMatrix = drawItemData._worldMatrix;
-    sharedData.bounds.GetMatrix().Get(worldMatrix.matrix);
```

Comment:
Moving this call inside if-statements fixes an issue with bound box dimensions when a display layer is switched to bound box mode in multi-viewport